File: src/main/java/at/scheinecker/intellij/coco/CocoLexerAdapter.java
package at.scheinecker.intellij.coco;

import com.intellij.lexer.FlexAdapter;

/**
 * @author Thomas Scheinecker <a href="mailto:tscheinecker@gmail.com">tscheinecker@gmail.com</a>
 */
public class CocoLexerAdapter extends FlexAdapter {
    public CocoLexerAdapter() {
        super(new CocoLexer());
    }
}

File: src/main/kotlin/me/salzinger/intellij/coco/CocoLexerAdapter.kt
package me.salzinger.intellij.coco

import com.intellij.lexer.FlexAdapter

/**
 * @author Thomas Salzinger [tsalzinger@gmail.com](mailto:tsalzinger@gmail.com)
 */
class CocoLexerAdapter : FlexAdapter(me.salzinger.intellij.coco.CocoLexer())
